Dockerfile

```diff
@@ -2,8 +2,10 @@ FROM tweekmonster/vim-testbed:latest
 
 RUN install_vim -tag v8.0.0027 -build \
                 -tag v8.1.0519 -build \
+                -tag v8.2.2401 -build \
                 -tag neovim:v0.2.0 -build \
-                -tag neovim:v0.3.5 -build
+                -tag neovim:v0.3.5 -build \
+                -tag neovim:v0.4.4 -build
```

Comment:
I think we can cut out the versions in the middle for Vim and NeoVim. Testing with the the oldest and newest supported versions should be enough to catch issues with functions being missing and so on.

Reply:
Got it, removed Vim 8.1 and Neovim 0.3.5